src/scala/io/bazel/rules_scala/jmh_support/BUILD

```diff
@@ -8,8 +8,6 @@ scala_library(
         "//external:io_bazel_rules_scala/dependency/jmh/jmh_core",
         "//external:io_bazel_rules_scala/dependency/jmh/jmh_generator_asm",
         "//external:io_bazel_rules_scala/dependency/jmh/jmh_generator_reflection",
-        "//external:io_bazel_rules_scala/dependency/jmh/org_ows2_asm_asm",
```

Comment:
should these be in the runtime_deps?

Reply:
Not sure, I'll take a closer look once I get tests passing. Still want to add some stuff to this PR that I noticed while working on one of our bigger repos at stripe. I'll make this WIP

Reply:
Well, I guess if tests pass, let's keep it simple.

Reply:
that said, we may not actually be running a benchmark in the tests, can we move to runtime_deps for now and open an issue to see if it is really needed?